glimr/search.py

```diff
@@ -120,6 +123,43 @@ def __init__(
         space["fit_kwargs"] = fit_kwargs
         space["loader"] = loader
         self._space = space
+        self.cv = cv
+
+        # if cv is 'True', check if search space contains cv parameters
+        if cv:
+            data_keys = list(space["data"].keys())
+            if "cv_fold_index" in data_keys and "cv_fold_index" in data_keys:
```

Comment:
See comment above. If the user passes the number of folds to search, then we can add these keys to the config in the proper way and not need to check them.